java-checks/src/test/java/org/sonar/java/checks/AssertionsCompletenessCheckTest.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.sonar.java.checks;
 
+import org.fest.assertions.Assertions;
 import org.junit.Test;
 import org.sonar.java.checks.verifier.JavaCheckVerifier;
 
 public class AssertionsCompletenessCheckTest {
-
   @Test
   public void test() {
+    Assertions.assertThat("").as("");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Complete the assertion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2970)